inspirehep/modules/authors/forms.py

```diff
@@ -274,13 +274,26 @@ class AuthorUpdateForm(INSPIREForm):
 
     """Author update form."""
 
-    # Hidden field to hold author id information
     bai = fields.StringField(
-        widget=HiddenInput()
```

Comment:
I'm doing it here https://github.com/Panos512/inspire-next/blob/a48fabae60ee52fa0a9da56af9b2fa2f3d143c09/inspirehep/modules/authors/views/holdingpen.py#L201 isn't this enough?